.gitignore

```diff
@@ -4,9 +4,13 @@
 *.dll
 *.so
 *.dylib
```

Comment:
I would just drop default content of `.gitignore`.
Also maybe we should talk about global `.gitignore` so we do not put lines like `Session.vim`, `.vscode`, `.idea` into every repository.